maven-plugin/src/main/java/com/spotify/missinglink/maven/CheckMojo.java

```diff
@@ -197,6 +217,17 @@ public void execute() throws MojoExecutionException, MojoFailureException {
               + Joiner.on(", ").join(ConflictCategory.values()));
     }
 
+    if (!ignoreDestinationPackages.isEmpty() && !targetDestinationPackages.isEmpty()) {
+      throw new MojoExecutionException(
+          "Either ignoreDestinationPackages or targetDestinationPackages can be set, "
+              + "but not both.");
+    }
+
+    if (!ignoreSourcePackages.isEmpty() && !targetSourcePackages.isEmpty()) {
+      throw new MojoExecutionException(
+          "Either ignoreSourcePackages or targetSourcePackages can be set, " + "but not both.");
+    }
```

Comment:
I wonder if there is another (annoying) case to check for - that ignoreDestinationPackages isn't set in concert with targetSourcePackages, and targetDestinationPackages set in concert with ignoreSourcePackages ? seems like it becomes combinatorial in number of cases to check

Reply:
yeah... I wasn't sure about that one because there isn't a technical reason why the user couldn't set both `ignoreDestinationPackages` and `targetSourcePackages` configurations, right? It would be an odd move but I don't know that they're actually in conflict

Reply:
I think its fine to avoid checking for that if it isn't really a conflict